main.c

```diff
@@ -183,7 +183,7 @@ int main(int argc, char *argv[])
 	}
 	libusb_set_debug(context, 3);
 
-	corsairlink_device_scanner(context);
+	int device_count = corsairlink_device_scanner(context);
```

Comment:
I'm sorry for not defining coding standards, but could you move the variable declaration to the top with the rest of the variables. Also, could you move the device_count to within the function, ie corsairlink_device_scanner(context, device_count). I intend to have the function returns for error codes.